src/api/token.py

```diff
@@ -2,16 +2,18 @@
 from webargs.flaskparser import parser
 
 from src import implementor as impl
+from src.api.decorators import token_required
 from src.schemas import model
 
 bp = Blueprint("token", __name__)
 
 
 @bp.route("/token", methods=["post"])
-def create_token():
+@token_required
+def create_token(token):
```

Comment:
Can you explain why a token is required to create a token? Or in other words, how would you create a token when you don't have one initially? 🤔

Reply:
Of course! Good question. Only admins should be able to create tokens. Therefore the first admin token has to be inserted into the database by hand 🙃